.gitattributes

```diff
@@ -14,3 +14,7 @@
 
 # See https://github.com/golang/go/issues/9281
 * -text
+
+vendor/** linguist-generated
+vendor/ linguist-generated
```

Comment:
I think this needs to have `linguist-generated=true` according to https://github.com/kubernetes/test-infra/blob/master/prow/gitattributes/gitattributes.go#L89